mithril-stm/src/schnorr_signature/mod.rs

```diff
@@ -0,0 +1,173 @@
+// TODO: Remove
+#![allow(dead_code)]
+
+use midnight_circuits::{
+    ecc::{hash_to_curve::HashToCurveGadget, native::EccChip},
+    hash::poseidon::PoseidonChip,
+    types::AssignedNative,
+};
+use midnight_curves::{
+    Fq as JubjubBase, Fr as JubjubScalar, JubjubAffine, JubjubExtended, JubjubSubgroup,
+};
+use sha2::{Digest, Sha256};
+
+use anyhow::{Result, anyhow};
```

Comment:
You should separate the module declaration from the imports and put the imports after. It is usually a good practice to create a new module to do that.